service/src/test/java/uk/gov/hmcts/reform/fpl/service/CaseValidatorServiceTest.java

```diff
@@ -301,10 +354,34 @@ private Allocation allocationProposal() {
             .build());
     }
 
+    private List<Element<Applicant>> applicantWithInvalidEmailAddress(final String emailAddress) {
+        return wrapElements(Applicant.builder()
+            .party(ApplicantParty.builder()
+                .email(EmailAddress.builder()
+                    .email(emailAddress)
+                    .build())
+                .build())
+            .build());
+    }
+
+    private List<Element<Respondent>> respondentWithInvalidEmailAddress(final String emailAddress) {
+        return wrapElements(Respondent.builder()
+            .party(RespondentParty.builder()
+                .email(EmailAddress.builder()
+                    .email(emailAddress)
+                    .build())
+                .build())
+            .build());
+    }
+
     private Solicitor solicitor() {
         return Solicitor.builder().name("fred").email("fred@fred.me").build();
     }
 
+    private Solicitor solicitorWithInvalidEmailAddress(final String emailAddress) {
+        return Solicitor.builder().name("fred").email(emailAddress).build();
+    }
```

Comment:
Why not just re-use the solicitor method and have an email as a param?

Reply:
yes, we could but needed minimal updates to the class.